src/python/torchdistx/slowmo/slowmo_optimizer.py

```diff
@@ -174,11 +174,8 @@ def load_state_dict(self, state_dict):
         self.slowmo_lr = state_dict.pop("slowmo_lr")
         self.averager.step = state_dict.pop("step")
         self._base_optim.load_state_dict(state_dict)
-        # check that base optimizer's learning rate is stored in param_groups
-        if not (self.param_groups and self.param_groups[0]["lr"]):
-            raise ValueError(
-                "Base optimizer does not have parameters or learning rate specified."
-            )
+        if not self.param_groups:
```

Comment:
why do we remove the learning rate check here?

Reply:
@rohan-varma  I've added proper checks in this version, i.e. since it is possible to have multiple parameter groups and each of them should have `lr`, every group should have a check. Before, only first parameter group had a check